### PR TITLE
Use brand theme

### PR DESCRIPTION
**Pull request by x4d3**
You can preview here: https://awesome-badger.pages.dev/

Specs were here: https://xd.adobe.com/view/0b738fc8-30f5-490a-849f-9f014545811b-5e20/

### CODE REVIEW

File: _sass/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+@font-face {
+  font-family: 'Sentinel-Light';
+  src: url('/assets/fonts/Sentinel-Light.otf') format('truetype');
+}
+
+@font-face {
+  font-family: 'ProximaNova-Bold';
+  src: url('/assets/fonts/322627_0_0.woff2') format('woff2'),
+  url('/assets/fonts/322627_0_0.woff') format('woff'), url('/assets/fonts/322627_0_0.ttf') format('truetype');
```
Comment (jbadavis) [inline _sass/fonts.scss:64]: Why aren't the Proxima font files called "ProximaNova.woff2` etc?

File: _sass/minima/_layout.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 .site-header {
   border-top: 2px solid $grey-color-dark;
-  border-bottom: 1px solid $grey-color-light;
+  border-bottom: 1px solid #CCCCCC;
```
Comment (jbadavis) [inline _sass/minima/_layout.scss:6]: Should this colour be a variable? It's used multiple times in this file.

File: _sass/minima/_layout.scss [hunk 3]
```diff
@@ -137,49 +131,51 @@
...
 
-
 /**
  * Page content
  */
 .page-content {
   padding: $spacing-unit 0;
   flex: 1;
+  @include media-query($on-laptop) {
+    padding: $spacing-unit/2 $spacing-unit/2;
```
Comment (jbadavis) [inline _sass/minima/_layout.scss:177]: Can this just be:

```suggestion
    padding: $spacing-unit / 2;
```

File: _sass/minima/_layout.scss [hunk 4]
```diff
@@ -195,19 +191,28 @@
   list-style: none;
 
   > li {
-    margin-bottom: $spacing-unit;
+    margin-bottom: 2* $spacing-unit;
```
Comment (jbadavis) [inline _sass/minima/_layout.scss:194]:
```suggestion
    margin-bottom: 2 * $spacing-unit;
```

File: _sass/fonts.scss (hunk 1 above)
Comment (x4d3) [reply _sass/fonts.scss:64]: I've copied from https://github.com/redbadger/website-honestly/ but I'll rename.

**Comment by x4d3** [comment]
@jbadavis Thanks for you review, I've addressed in https://github.com/redbadger/awesome-badger/pull/70/commits/d1d518977e2416c28c01697132edf08cd2e42ebb

File: _sass/minima/_base.scss [hunk 5]
```diff
@@ -239,20 +223,35 @@ table {
...
+.masthead{
+  line-height: 1.2;
+  font-family: 'Sentinel-Bold';
+  margin: 30px 0px 80px 0px;
+  font-size: 50px;
+  @include media-query($on-laptop) {
+    font-size: 25px;
+    margin: 0px 0px 20px 0px;
+  }
+}
```
Comment (jbadavis) [inline _sass/minima/_base.scss:257]: Missing new line.

File: _sass/minima/_base.scss [hunk 6]
```diff
@@ -239,20 +223,35 @@ table {
...
+
   td {
     border: 1px solid $grey-color-light;
   }
 }
+
+.masthead{
+  line-height: 1.2;
+  font-family: 'Sentinel-Bold';
+  margin: 30px 0px 80px 0px;
```
Comment (jbadavis) [inline _sass/minima/_base.scss:251]:
```suggestion
  margin: 30px 0px 80px;
```

File: _sass/minima/_base.scss [hunk 7]
```diff
@@ -239,20 +223,35 @@ table {
...
+  font-size: 50px;
+  @include media-query($on-laptop) {
+    font-size: 25px;
+    margin: 0px 0px 20px 0px;
```
Comment (jbadavis) [inline _sass/minima/_base.scss:255]:
```suggestion
    margin: 0px 0px 20px;
```

**Review by jbadavis — APPROVED** [review]
🚀

**Pull request merged**